control-plane/pkg/reconciler/broker/controller.go

```diff
@@ -97,6 +100,10 @@ func NewController(ctx context.Context, watcher configmap.Watcher, configs *Conf
 
 	impl := brokerreconciler.NewImpl(ctx, reconciler, kafka.BrokerClass)
 
+	reconciler.EnqueueAfter = func(broker *eventing.Broker, duration time.Duration) {
+		impl.EnqueueAfter(broker, duration)
```

Comment:
can you just use the function pointer here? :smile:

Reply:
I was trying to prevent passing a random object that is not a broker since `impl.EnqueueAfter` takes `interface{}`.

Reply:
prevent who from passing a random object?

Reply:
We developers of this reconciler, I like having compile-type checks over raw `interface{}`.

Once we had a bug in eventing because we were enqueuing a trigger in place of a broker in the broker reconciler.